apps/storybook/src/Table.stories.tsx

```diff
@@ -2422,6 +2422,75 @@ Virtualized.argTypes = {
   data: { control: { disable: true } },
 };
 
+export const ScrollToItem: Story<Partial<TableProps>> = (args) => {
+  const onClickHandler = (
+    props: CellProps<{ name: string; description: string }>,
+  ) => action(props.row.original.name)();
+
+  const columns = useMemo(
+    () => [
+      {
+        Header: 'Table',
+        columns: [
+          {
+            id: 'name',
+            Header: 'Name',
+            accessor: 'name',
+          },
+          {
+            id: 'description',
+            Header: 'Description',
+            accessor: 'description',
+          },
+          {
+            id: 'click-me',
+            Header: 'Click',
+            width: 100,
+            Cell: (props: CellProps<{ name: string; description: string }>) => {
+              const onClick = () => onClickHandler(props);
+              return <Anchor onClick={onClick}>Click me!</Anchor>;
+            },
+          },
+        ],
+      },
+    ],
+    [],
+  );
+
+  const data = useMemo(() => {
+    const size = 100000;
+    const arr = new Array(size);
+    for (let i = 0; i < size; ++i) {
+      arr[i] = {
+        id: i,
+        name: `Name${i}${i === 12345 ? ' - Scrolled to me!' : ''}`,
+        description: `Description${i}${
+          i === 12345 ? ' - Scrolled to me!' : ''
+        }`,
+      };
+    }
+    return arr;
+  }, []);
+
+  return (
+    <Table
+      enableVirtualization
+      columns={columns}
+      emptyTableContent='No data.'
+      {...args}
+      style={{ maxHeight: '90vh' }}
+      data={data}
+      getRowId={(row) => row.id}
+      scrollToRow={(rows) => rows.findIndex((row) => row.id === data[12345].id)}
```

Comment:
Not sure whether to show this way as an example or better `(rows) => rows.findIndex(row => row.original === data[12345])`